build.gradle

```diff
@@ -1,23 +1,12 @@
-buildscript {
-    repositories {
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4'
-        classpath 'org.ajoberstar:gradle-git:1.4.2'
-    }
+plugins {
+    id "groovy"
+    id "java-gradle-plugin"
+    id "com.jfrog.bintray" version "1.8.4"
+    id "maven-publish"
```

Comment:
I believe that the Maven Publish plugin is already applied by the Bintray plugin if I am not mistaken.

Reply:
I will remove it but If i havent done it tomorrow then it probably will jave to wait 2 weeks